src/Playwright/Transport/Protocol/Generated/PlaywrightInitializer.cs

```diff
@@ -45,7 +45,7 @@ internal class PlaywrightInitializer
     public List<DeviceDescriptorEntry> DeviceDescriptors { get; set; }
 
     [JsonPropertyName("selectors")]
-    public Core.Selectors Selectors { get; set; }
+    public Core.SelectorsOwner Selectors { get; set; }
```

Comment:
This will get overridden in the next protocol roll, need to think how we should do it. Maybe just a hacky if inside our Generator.

Reply:
We can just rename `Selectors` to `SelectorsAPI` and `SelectorsOwner` to `Selectors`.